core/kazoo_schemas/src/kz_json_schema.erl

```diff
@@ -768,35 +771,35 @@ depreciated_validation_error(<<"account">>, <<"expired">>, Message, Options) ->
     build_validate_error([<<"account">>]
                         ,<<"expired">>
                         ,Message
-                        ,props:insert_values([{'error_code', 423}
-                                             ,{'error_message', <<"locked">>}
-                                             ]
-                                            ,Options
-                                            )
+                        ,props:set_values([{'error_code', 423}
+                                          ,{'error_message', <<"locked">>}
+                                          ]
+                                         ,Options
+                                         )
                         );
 depreciated_validation_error(<<"account">>, <<"disabled">>, Message, Options) ->
     build_validate_error([<<"account">>]
                         ,<<"disabled">>
                         ,Message
-                        ,props:insert_values([{'error_code', 423}
```

Comment:
why is this changing? insert_values does not override values

Reply:
We had a talk at one of our stand up sessions and we agreed to let it be. It is a fix for a bug.